contrib/rbac-setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh --
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SVCCAT_NAMESPACE=catalog
+kubectl create namespace ${SVCCAT_NAMESPACE}
+kubectl create serviceaccount server --namespace=${SVCCAT_NAMESPACE}
+kubectl create serviceaccount controller --namespace=${SVCCAT_NAMESPACE}
+kubectl create -f svc-cat-rbac.yaml
```

Comment:
This assumes we're running this file while in `contrib/`. I'd add

```
ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

to the top of the file, and then change the file reference to `"${ROOT}/contrib/svc-cat-rbac.yaml`